core/src/packet.rs

```diff
@@ -62,7 +63,6 @@ where
         let user_data;
         unsafe {
             user_data = U::from_raw_const_ptr(this.raw().user_data.cast_const().cast());
-            sys::tb_client_release_packet(this.handle.raw, this.raw);
```

Comment:
@50U10FCA7 shouldn't this be `Box::from_raw(this.raw)` too?

Reply:
@tyranron `Packet` will be dropped in this method (and `Box::from_raw(this.raw)` will be called) because it captures `self`.